bashfile.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
```

Comment:
I'm seeing bashfile.sh permission denied currently but I fixed this by running chmod u+x bashscript.sh in terminal